frank-doc-doclet/src/main/java/org/frankframework/frankdoc/Utils.java

```diff
@@ -270,7 +270,7 @@ private static String getLinkReplacement(String linkBody) {
 	}
 
 	public static String replaceClassFieldValue(String text, FrankClass context) throws FrankDocException {
-		return replacePattern(text, JAVADOC_VALUE_START_DELIMITER, JAVADOC_SUBSTITUTION_PATTERN_STOP_DELIMITER, s -> getClassFieldValueReplacement(s, context));
+		return replacePattern(text, JAVADOC_VALUE_START_DELIMITER, JAVADOC_SUBSTITUTION_PATTERN_STOP_DELIMITER, s -> "<code>" + getClassFieldValueReplacement(s, context) + "</code>");
```

Comment:
Zo te zien is dit de enige plek waar `getClassFieldValueReplacement` gebruikt wordt. Ik zou het dan logischer vinden om in die method (bv op regel 299) het resultaat te wrappen met code tags